ED/protocol.html

```diff
@@ -330,12 +330,12 @@ <h2 id="table-of-contents">Table of Contents</h2>
                   </ol>
                 </li>
                 <li class="tocline">
-                  <a class="tocxref" href="#notifications"><span class="secno">6</span> <span class="content">Notifications</span></a>
+                  <a class="tocxref" href="#notifications"><span class="secno">6</span> <span class="content">Linked Data Notifications</span></a>
```

Comment:
```suggestion
                  <a class="tocxref" href="#linked-data-notifications"><span class="secno">6</span> <span class="content">Linked Data Notifications</span></a>
```

If accepted, https://github.com/solid/specification/pull/409/files#r932456895 should also be accepted.

Reply:
I support the changes, but I think it could just as well be done in a separate PR.